src/init.cpp

```diff
@@ -1392,7 +1382,9 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         .chainparams = chainparams,
         .adjusted_time_callback = GetAdjustedTime,
     };
-    ApplyArgsManOptions(args, chainman_opts);
+    if (const auto err{ApplyArgsManOptions(args, chainman_opts)}) {
```

Comment:
In commit "Move ::nMinimumChainWork into ChainstateManager" (fa02eb1354ff16e91ab6b9bdb362be0de24f1750)

Maybe note slight change in behavior here in commit message: "Invalid non-hex" error will now be triggered later, after `bitcoind` has daemonized instead of before it daemonizes.

Reply:
Thanks, but fixed in another way